### ISSUE TITLE
[nri-bundle] helm uninstall fails

### ISSUE THREAD

**Issue by luisdavim** [newrelic/helm-charts#667]
Bug description

Running helm uninstall fails

Version of Helm and Kubernetes

```console
helm version
version.BuildInfo{Version:"v3.7.2", GitCommit:"663a896f4a815053445eec4153677ddc24a0a361", GitTreeState:"clean", GoVersion:"go1.16.10"}
```
```console
kubectl version
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.2", GitCommit:"092fbfbf53427de67cac1e9fa54aaa09a28371d7", GitTreeState:"clean", BuildDate:"2021-06-16T12:59:11Z", GoVersion:"go1.16.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.11-eks-f17b81", GitCommit:"f17b810c9e5a82200d28b6210b458497ddfcf31b", GitTreeState:"clean", BuildDate:"2021-10-15T21:46:21Z", GoVersion:"go1.15.15", Compiler:"gc", Platform:"linux/amd64"}
```
Which chart?

The chart is nri-bundle-3.3.0

What happened?

The helm uninstall fails, waiting for the `vizier-deleter` job to finish.
In the events for the job I see:

```console
Events:
  Type     Reason        Age                   From            Message
  ----     ------        ----                  ----            -------
  Warning  FailedCreate  2m58s (x6 over 8m8s)  job-controller  Error creating: pods "vizier-deleter-" is forbidden: error looking up service account newrelic/updater-service-account: serviceaccount "updater-service-account" not found
```

What you expected to happen?

I would expect the chart to be uninstalled successfully

How to reproduce it?

Just a normal helm uninstall as mentioned in the readme.

I'm using the same setup as described in https://github.com/newrelic/helm-charts/issues/661

**Comment by kang-makes** [comment]
Fixed with #683

**Comment by luisdavim** [comment]
Hi, with `nri-bundle-3.4.0` the `vizier-deleter` does start now but it just stays running until helm times out, how long is it supposed to take?
So different issue but same end result.
